FilesAndExceptions_task8/src/main/java/org/Marta/PeselValidator.java

```diff
@@ -0,0 +1,18 @@
+package org.Marta;
+
+public class PeselValidator {
+    public static void validatePesel(Object peselNumber) throws IllegalLengthException, WrongTypeOfDataException {
```

Comment:
(załóż, że argument jest typu String)

Reply:
unikaj exceptionów checked. Checked exceptiony to zło i zaszłość javy.

Tylko unchecked. 

https://phauer.com/2015/checked-exceptions-are-evil/